### PR TITLE
DEVDOCS-2013-IA refactor edits

### PR DESCRIPTION
**Pull request by mikrdgz**
# [DEVDOCS-2013](https://jira.bigcommerce.com/browse/DEVDOCS-2013)

## What changed?
IA Refactor edits

### CODE REVIEW

**Comment by mikrdgz** [comment]
@uwikaiddi ready for review

**Comment by uwikaiddi** [comment]
@mikrdgz, this is the first time I've seen a page done entirely in HTML. Do you know why this one is different from the others?

**Comment by mikrdgz** [comment]
@uwikaiddi sorry, one of my VS code tools creates an HTML page in addition to the MD page and I forgot to manually remove it. Just commit the changes and I will delete the unneeded html page from the repo

File: docs/api-docs/getting-started/rest-api-authentication.md [hunk 1]
```diff
@@ -157,51 +161,58 @@ During the app installation process, your app will use the Client Id and Client
...
+> As of July 31, 2018, new BigCommerce stores are no longer able to create legacy API Accounts (accounts using HTTP basic auth) within their control panels. Existing legacy API Accounts will continue to work until further notice, but we strongly recommend migrating to OAuth as soon as possible.
 
 </div>
 </div>
 </div>
 
-### Migrating to OAuth comes with several benefits:
+### Benefits of migrating to OAuth:
+
+It is recommended you migrate from using legacy API credentials to OAuth. Some benefits include:
```
Comment (uwikaiddi) [inline docs/api-docs/getting-started/rest-api-authentication.md:172]:
```suggestion
It is recommended you migrate from using legacy API credentials to OAuth. Some benefits include the following:
```

**Comment by uwikaiddi** [comment]
@mikrdgz, thanks for the clarification. I went ahead and deleted the HTML file in this PR. Also included some corrections for the Markdown file.

**Pull request merged**